src/microbe_stage/membrane_system.h

```diff
@@ -53,6 +53,10 @@ class MembraneComponent : public Leviathan::Component {
     void
         setColour(const Float4& value);
 
+    //! Should set the health percentage.
+    void
+        setHealthPercentage(float value);
```

Comment:
The health value is between 0 and 1. Percentages go up to 100. This should be renamed to like health fraction and a comment should be added about the valid range.